lookout/style/typos/cmdline_tools.py

```diff
@@ -0,0 +1,291 @@
+        help="Path to a FastText model dump." + addition)
+
+
+def create_parser() -> ArgumentParser:
+    """
+    Create a parser for the lookout.style.typos utility.
+
+    :return: an ArgumentParser with an handler defined in the handler attribute.
+    """
+    # Deferred imports to speed up loading __init__
+    from lookout.style.typos.train_test_corrector import (get_train_test, prepare_data, test,
+                                                          train, train_from_scratch)
+
+    parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+
+    # General options
+    parser.add("--log-level", default="DEBUG", help="Log verbosity level.")
+
+    subparsers = parser.add_subparsers(help="Commands")
+
+    def add_parser(name, help):
+        return subparsers.add_parser(
+            name, help=help, formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+
+    # Prepare raw data for corrector
+    prepare_parser = add_parser("prepare-data", "Prepare raw dataset for corrector training.")
+    prepare_parser.set_defaults(handler=prepare_data)
+    add_input_path_arg(prepare_parser)
+    add_frequency_column_arg(prepare_parser)
+    add_vocabulary_size_arg(prepare_parser)
+    add_frequencies_size_arg(prepare_parser)
+    add_vocabulary_path_arg(prepare_parser)
+    add_frequencies_path_arg(prepare_parser)
+    add_save_prepared_path_arg(prepare_parser)
+
+    # Get train and test datasets
+    train_test_parser = add_parser("get-train-test",
+                                   "Get train and test datasets with artificially"
+                                   "created typos from prepared data.")
+    train_test_parser.set_defaults(handler=get_train_test)
+    train_test_parser.add_argument("prepared_data", metavar="data", type=str,
+                                   help=".csv dump of dataframe correct splitted identifiers"
+                                        ". Must contain columns 'token_split' and 'token'.")
+    add_train_size_arg(train_test_parser)
+    add_test_size_arg(train_test_parser)
+    add_frequency_column_arg(train_test_parser)
+    add_save_train_path_arg(train_test_parser)
+    add_save_test_path_arg(train_test_parser)
+
+    # Train TyposCorrector model.
+    train_parser = add_parser("train",
+                              "Create and train TyposCorrector model on given data.")
+    train_parser.set_defaults(handler=train)
+    train_parser.add_argument("train_data", metavar="data", type=str,
+                              help=".csv dump of a train dataframe, containing"
+                                   "columns'token' and 'correct_token'.")
+    add_vocabulary_path_arg(train_parser)
+    add_frequencies_path_arg(train_parser)
+    add_fasttext_path_arg(train_parser)
+    add_threads_number_arg(train_parser)
+    add_save_model_path_arg(train_parser)
+
+    # Evaluate on different styles dataset
+    test_parser = add_parser("test",
```

Comment:
Testing must be performed straight after the training. The reports are included into the model file, see https://docs.sourced.tech/modelforge/model#metrics

This does not mean that this command should be deleted. This means that we call it's handler after training in the previous command, and require a test dataset there.